svgsynoptic2/taurussynopticwidget.py

```diff
@@ -151,6 +152,16 @@ def filter_fragment(self, model, value):
                 pass
         return value
 
+    def set_custom_value(self, model, attr_value):
+        # this is the special case where value is modified for PyAlarm class
+        # when alarm is trigered PyAlarm attribute returns 'True' value, icon on synoptic is green
+        # set_custom_value change value for better alarms visualisation
+        # function can be customised for any other tango device classes
+        if 'PyAlarm' in PyTango.get_device_proxy(model).info().dev_class:
```

Comment:
Maybe this is better to keep only one return statement:

```python
dev_class = PyTango.get_device_proxy(model).info().dev_class
return not(attr_value) if 'PyAlarm' in dev_class else attr_value
```
But it is less "customized friendly"